validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -79,25 +118,51 @@ def __init__(self, block_cache, state_view, data_dir):
         self._state_view = state_view
 
     def verify_block(self, block_state):
-        return True
+        if block_state._block_header.consensus == b"Devmode":
+            return True
+        else:
+            return block_state._block_header.consensus == b"Devmode"
```

Comment:
user the header property off of the BlockWrapper.

    @property
    def header(self):
        """
        Returns the header of the block
        """
        if self._block_header is None:
            self._block_header = BlockHeader()
            self._block_header.ParseFromString(self.block.header)
        return self._block_header